scripts/scenarios/cloud7/qa/no-ssl/qa-scenario-2c.yaml

```diff
@@ -69,6 +69,7 @@ proposals:
 - barclamp: ceph
   attributes:
     disk_mode: first
+    client_network: admin
```

Comment:
Yes of course, thats the whole point of it.. mons need to be listening on the public network